Source/Core/DolphinQt2/CMakeLists.txt

```diff
@@ -12,7 +12,11 @@ set(SRCS
 	Resources.cpp
 	Settings.cpp
 	ToolBar.cpp
+	Util.cpp
```

Comment:
I really don't like Util files. I would greatly appreciate it if you find a more suitable home for `FormatSize`.